src/hooks/useTxBatchTrades.tsx

```diff
@@ -60,17 +60,19 @@ export function useTxBatchTrades(
   const [error, setError] = useState('')
   const [txBatchTrades, setTxBatchTrades] = useState<TxBatchTrades>({ trades: [], transfers: [] })
   const [accounts, setAccounts] = useState<Accounts>()
-  const txOrders = usePrevious(JSON.stringify(orders?.map((o) => ({ owner: o.owner, kind: o.kind, receiver: o.receiver })))) // We need to do a deep comparison here to avoid useEffect to be called twice (Orders array is populated partially from different places)
+  const txOrders = usePrevious(
+    JSON.stringify(orders?.map((o) => ({ owner: o.owner, kind: o.kind, receiver: o.receiver }))),
```

Comment:
@matextrem I forgot to tell you that I had to make some changes [here](https://github.com/cowprotocol/explorer/pull/137/commits/bbe7cea6aab161cc52796d6fc3e6717c0dbcdda0) to comply with the linter.